storage/innobase/log/log0recv.cc

```diff
@@ -3962,11 +3962,7 @@ recv_group_scan_log_recs(
 	bool		last_phase)
 {
 	DBUG_ENTER("recv_group_scan_log_recs");
-<<<<<<< HEAD
-//	DBUG_ASSERT(!last_phase || recv_sys->mlog_checkpoint_lsn > 0);
-=======
-	assert(!last_phase || recv_sys->mlog_checkpoint_lsn > 0);
->>>>>>> mysql-5.7.34
+//	assert(!last_phase || recv_sys->mlog_checkpoint_lsn > 0);
```

Comment:
can this be uncommented? (HEAD enabled this assert)

Reply:
This is currently commented out in 2.4 code - https://github.com/percona/percona-xtrabackup/blob/2.4/storage/innobase/log/log0recv.cc#L3965